### ISSUE TITLE
Regarding training/validation accuracy

### ISSUE THREAD

**Issue by ChengFu0118** [YijinHuang/pytorch-DR#4]
Hi Yijin,

Thanks so much for sharing your code! I successfully run your code on my server. Yet, I don't know why the training/validation accuracy stuck at ~20% and never go up. The training accuracy reaches 30% while the validation accuracy drops to 10% after 20 epochs. Do you experience the same issue? Or it is a common observation for this regression problem?

**Comment by YijinHuang** [comment]
I didn't experience this issue. Is the small net using SMALL_NET_CONFIG you are training? The performance of the small network may be not stable.  How about the final large net?
